admin/banner_manager.php

```diff
@@ -130,7 +130,7 @@
 
           $sql = "UPDATE " . TABLE_BANNERS . "
                   SET
-                    date_scheduled = :scheduledDate,
+                    date_scheduled = DATE_ADD(:scheduledDate, INTERVAL '00:00:00' HOUR_SECOND),

```

Comment:
Doesn't this need something more, for when `$date_scheduled` is blank?

Something similar to line 140? `if ($date_scheduled != '') {`